src/layout/List/ListComponent.tsx

```diff
@@ -75,46 +85,124 @@ export const ListComponent = ({ node }: IListProps) => {
     return JSON.stringify(selectedRow) === JSON.stringify(row);
   }
 
+  function isRowChecked(row: Row): boolean {
+    return (formData?.saveToList as Row[]).some((selectedRow) => {
+      const { altinnRowId, ...rest } = selectedRow;
```

Comment:
```suggestion
      const { altinnRowId: _, ...rest } = selectedRow;
```

This gets rid of the eslint error 🥳